### ISSUE TITLE
Insert case/of completion not working

### ISSUE THREAD

**Issue by sentience** [halohalospecial/atom-elmjutsu#72]
The ‘Insert case/of’ special completion is not being offered to me, even though it is enabled in my preferences. Other special completions are working fine.

Atom 1.15.0 stable

**Comment by halohalospecial** [comment]
Hi!  Do you have a sample scenario where it's not working?  Note that there should be a space at the end of the line for it to offer case/of completion :-)  You can try this example (type space after "case bool"):

```
module Main exposing (..)

foo : Bool -> Int
foo bool =
    case bool
```

![case-of-example](https://cloud.githubusercontent.com/assets/2138906/24530078/2cd8a42a-15e2-11e7-8b3b-a9a37625fe3f.gif)

**Comment by sentience** [comment]
Tried it with the exact code above, and the completion is not offered. I made sure to type the space after `bool`.

Not sure what you mean by “only works if there are type signatures”. There is a type signature for the `foo` function in the example above; is that all you mean? Or is this some other feature of Elmjutsu that must be enabled (such as the type tooltips)?

**Comment by halohalospecial** [comment]
Yep.  Your case sure is weird hmm.  That should work if the other special completions are working fine.  Can you check if there are errors in the Atom console?

**Comment by sentience** [comment]
Nothing in the console, I’m afraid.

Aha! I disabled the case/of special completion option, closed Preferences, reopened preferences, enabled the case/of special completion option, and it’s working now! ¯\_(ツ)_/¯

**Comment by halohalospecial** [comment]
Haha!  Software :grinning: Glad you got it working!

**Comment by jacoporicare** [comment]
Hello, I'm experiencing the same issue and I'm not able to fix it 🙁. No errors in console, nothing.
I'm on MacBook Pro, late 2015, macOS Sierra 10.12.6.

**Comment by halohalospecial** [comment]
Hi @jacoporicare, can you provide a screenshot?  Thanks!

**Comment by jacoporicare** [comment]
![case-of](https://user-images.githubusercontent.com/18525854/30002234-57fb0534-90a4-11e7-9932-4d6a5f23c99d.gif)

**Comment by jacoporicare** [comment]
BTW I know it's a different topic but I have to press Enter twice when I type `=` because I have to "confirm" the autocomplete. Is there a way to not pre-select the first item or even better not to offer an item which is exactly the same like I typed? E.g. if I type `=` then it's not necessary to offer `=`.

**Comment by halohalospecial** [comment]
@jacoporicare, I tried your example and it's working.  Settings view also looks fine.  Maybe another Atom package has a bad interaction with that feature?

Regarding the issue with `=`, the hack discussed [here](https://github.com/halohalospecial/atom-elmjutsu/issues/5) doesn't work anymore with the recent version of `autocomplete-plus`. :cry:  As a workaround, you can set `Autocomplete Min Chars` to `2` or higher if you don't want to autocomplete 1-character symbols.

**Comment by jacoporicare** [comment]
Thank you, I'll investigate it further. Thanks for the tip with `autocomplete` settings, that should help.
